tools/v2fix/_stage/ddtracetypes/ddtracetypes.go.golden

```diff
@@ -9,8 +9,7 @@ package main
 import (
 	"time"
 
-	"github.com/DataDog/dd-trace-go/v2/ddtrace"
-	"github.com/DataDog/dd-trace-go/v2/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
```

Comment:
Hmmm, shouldn't this tool change the imports to v2?

Reply:
@kakkoyun In the tests we only run each probe independently. That's why the import doesn't change. Combined they change everything together.